android/src/test/java/com/amplitude/android/AmplitudeTest.kt

```diff
@@ -3,19 +3,29 @@ package com.amplitude.android
 import android.app.Application
 import android.content.Context
 import com.amplitude.android.plugins.AndroidLifecyclePlugin
+import com.amplitude.common.android.AndroidContextProvider
+import com.amplitude.core.events.BaseEvent
+import com.amplitude.core.platform.EventPlugin
+import com.amplitude.core.platform.Plugin
 import com.amplitude.core.utilities.InMemoryStorageProvider
 import com.amplitude.id.IMIdentityStorageProvider
 import com.amplitude.id.IdentityConfiguration
 import com.amplitude.id.IdentityContainer
-import io.mockk.mockk
-import io.mockk.mockkStatic
+import io.mockk.*
```

Comment:
Very minor, maybe we can use explicit imports instead of `*`, see [this post](https://discuss.kotlinlang.org/t/star-import-vs-importing-each-element/14460). This behavior can be configured in IDE, see [this post](https://stackoverflow.com/questions/48755419/disable-intellij-kotlin-starred-imports).